### ISSUE TITLE
Add hidden h1

### ISSUE THREAD

**Issue by tspeed90** [WageCheck/workerbird-website#13]
For accessibility purposes, we need to have an h1 to describe the page. Doesn't need to be visible, but should be included in the DOM. Shouldn't be set to display:hidden or will be hidden from AOM as well.

Also include a skip link?
